test/specs/components/Popup/Popup-test.tsx

```diff
@@ -102,4 +111,28 @@ describe('Popup', () => {
       expect(yOffsetTransformed.trim()).toBe(yOffset)
     })
   })
+
+  describe('toggle popup', () => {
+    test('with enter/space key', () => {
```

Comment:
I think this test is limited to testing that Enter on trigger opens the popup and Space on trigger closes the popup.

Maybe make 2 separate tests. One for Enter one for Space. Same workflow. Test that key opens the popup and then the same key closes it.

You can even use the same function and create 2 tests with it, just change the key, pass it as parameter.

Reply:
good point :) implemented two tests one for Enter and second for Space